qr-code-auth/README.md

```diff
@@ -0,0 +1,11 @@
+# QrCodeAuth
+
+```
+{
+  'user_id': string,
+  'device_info':string,
+  'is_authorized': boolean,
+  'access_token': string,
```

Comment:
What is access_token here?

Reply:
Access token to authenticate the user on API calls

Reply:
@shreya-mishra , will this data be sent from mobile to backend or from backend to mobile?
I ask this because, in case of mobile to backend, access_token and is_authorized don't make sense to me.
And in case of backend to mobile, user_id and device_info don't make sense to me.
Could you please explain where this model will fit in the flow?

Reply:
In that case could you please explain why backend is sending user id and device info back to mobile?

Reply:
> In that case could you please explain why backend is sending user id and device info back to mobile?

@prakashchoudhary07 do we need to store the access token in the table? Or will it be generate via github?

Reply:
> > In that case could you please explain why backend is sending user id and device info back to mobile?
> 
> @prakashchoudhary07 do we need to store the access token in the table? Or will it be generated via Git Hub?

Reply:
What access tokens are we talking about here?
Is the login related to Github?
 Why will GitHub give access tokens when login in with QR?